tools/pipelines/templates/include-test-perf-benchmarks.yml

```diff
@@ -27,20 +27,19 @@ jobs:
     pool: ${{ parameters.poolBuild }}
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
     variables:
-    - group: ado-feeds
     - name: isTestBranch
       value: ${{ startsWith(variables['Build.SourceBranch'], 'refs/heads/test/') }}
       readonly: true
     - name: feed
       ${{ if eq(variables.isTestBranch, 'true') }}:
-        value: $(internalFeed) # Comes from the ado-feeds variable group
+        value: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/internal/npm/registry/
       ${{ else }}:
-        value: $(buildFeed) # Comes from the ado-feeds variable group
+        value: https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/
     - name: devFeed
       ${{ if eq(variables.isTestBranch, 'true') }}:
-        value: $(internalFeed) # Comes from the ado-feeds variable group
```

Comment:
problem is that this declaration of devFeed is self-referential, when it's meant to refer to the variable from `ado-feeds`.

Reply:
well, the one on L43. put the comment on the wrong line :p